internal/usecases/cli/cli_test.go

```diff
@@ -92,7 +92,7 @@ func TestValidateConfigs(t *testing.T) {
 
 		err := useCases.ValidateConfigs(config)
 		assert.Error(t, err)
-		assert.Equal(t, "jSONOutputFilePath: JSON File path is required or is invalid: is not valid .json file.",
+		assert.Equal(t, "jSONOutputFilePath: Output File path is required or is invalid: not valid file of type .json.",
```

Comment:
Those tests scenarios are all about JSON output files.
I think it would be nice to cover situations of TEXT output files.
Something like:
```go
t.Run("Should return error when the text output file is invalid", func(t *testing.T) {
	config := &cliConfig.Config{}
	config.SetWorkDir(&workdir.WorkDir{})
	config.NewConfigsFromEnvironments()
	config.SetPrintOutputType(outputtype.Text)
	config.SetJSONOutputFilePath("test.test")

	err := useCases.ValidateConfigs(config)
	assert.Error(t, err)
	assert.EqualError(t, err, "jSONOutputFilePath: Output File path is required or is invalid: not valid file of type .txt.")
})
```
and
```go
t.Run("Should not return error when the text output file is valid", func(t *testing.T) {
	config := &cliConfig.Config{}
	config.SetWorkDir(&workdir.WorkDir{})
	config.NewConfigsFromEnvironments()
	config.SetPrintOutputType(outputtype.Text)
	config.SetJSONOutputFilePath("test.txt")

	err := useCases.ValidateConfigs(config)
	assert.NoError(t, err)
})
```